pkg/util/deployment/deployment.go

```diff
@@ -251,6 +252,10 @@ func labelPodsWithHash(podList *api.PodList, c clientset.Interface, namespace, h
 			if _, err := updatePodWithRetries(c.Core().Pods(namespace), &pod, func(podToUpdate *api.Pod) {
 				podToUpdate.Labels = labelsutil.AddLabel(podToUpdate.Labels, extensions.DefaultDeploymentUniqueLabelKey, hash)
 			}); err != nil {
+				if errors.IsNotFound(err) {
```

Comment:
Done and changed to ignore the error inside `updatePodWithRetries` and `updateRSWithRetries`. PTAL